docs/units/quantity.rst

```diff
@@ -426,13 +441,12 @@ units:
 Representing Vectors with Units
 ===============================
 
-|Quantity| objects can, like ``numpy`` arrays, be used to represent vectors or
-matrices by assigning specific dimensions to represent the coordinates or
-matrix elements, but that implies tracking those dimensions carefully. For
-vectors, you can use instead the representations underlying coordinates, which
+|Quantity| objects can, like ``numpy`` arrays, be used to represent vectors by
+assigning specific dimensions to represent the coordinates, as described in
```

Comment:
This ", as described" part is not true. Dimensions here refers to a specific axis of an array - maybe I should have written this as such in the first place! In any case, the link should not be here...